### PR TITLE
Draft Blog: Building a Development Environment for Cloud Engineering

### PR DESCRIPTION
**Pull request by spara**
101 level article to guide a new user to configuring their dev environment step-by-step

### CODE REVIEW

File: content/blog/configuring-your-dev-environment/index.md [hunk 1]
```diff
@@ -0,0 +1,681 @@
...
+
+{{% /choosable %}}
+{{% choosable os linux %}}
+Use brew to install golang.
+
+```bash
+$ brew install golang
+```
+
+It is a best practice to create a local programming environment and set `GOPATH` as an environment variable,  although it is not required for golang versions after 1.8. Pulumi creates a new local environment and `go.mod` to work with modules each time you start a new Project with `pulumi new`.
```
Comment (davidwrede) [inline content/blog/configuring-your-dev-environment/index.md:502]:
```suggestion
It is a best practice to create a local programming environment and set `GOPATH` as an environment variable, although it is not required for golang versions after 1.8. Pulumi creates a new local environment and `go.mod` to work with modules each time you start a new Project with `pulumi new`.
```

File: content/blog/configuring-your-dev-environment/index.md [hunk 2]
```diff
@@ -0,0 +1,681 @@
...
+
+{{< /choosable >}}
+
+{{< /chooser >}}
+
+## Next steps
+
+Congratulations! You have a fully configured environment, and you're ready to jump into cloud engineering. You can maintain your development environment using the package manager to add or update your toolset. Your code editor provides a modern development platform that takes advantage of all the advances in software engineering. Your coding experience will be more productive and less frustrating.
+
+What are the next steps? Begin with Pulumi's [Getting Started]({{< relref "/docs/get-started" >}}). You can skip the configuration sections and jump straight into your [first project]({{< relref "/docs/get-started/aws/create-project" >}}). Once you're done with your first project, try out example projects on [Github](https://github.com/pulumi/examples). You can start with simple projects with the `pulumi` client, such as deploying a web server on AWS with python.
```
Comment (davidwrede) [inline content/blog/configuring-your-dev-environment/index.md:669]:
```suggestion
What are the next steps? Begin with Pulumi's [Getting Started]({{< relref "/docs/get-started" >}}). You can skip the configuration sections and jump straight into your [first project]({{< relref "/docs/get-started/aws/create-project" >}}). Once you're done with your first project, try out example projects on [Github](https://github.com/pulumi/examples). You can start with simple projects with the `pulumi` client, such as deploying a web server on AWS with python.
```
```suggestion
What are the next steps? Begin with Pulumi's [Getting Started]({{< relref "/docs/get-started" >}}). You can skip the configuration sections and jump straight into your [first project]({{< relref "/docs/get-started/aws/create-project" >}}). Once you're done with your first project, try out example projects on [Github](https://github.com/pulumi/examples). You can start with simple projects using the `pulumi` CLI, such as deploying a web server on AWS with python.
```

**Review by davidwrede — COMMENTED** [review]
Looks good! I only had little nits in my feedback.

File: content/blog/configuring-your-dev-environment/index.md [hunk 3]
```diff
@@ -0,0 +1,681 @@
...
+- **Code editor:** This is personal preference but make sure that it can perform code completion, error checking, and use enums. These features can be the difference between hitting `Tab` and searching through online documentation for a function.
+- **Pulumi:** You can install Pulumi with a package manager and configure
+
+## Decisions
+
+Before proceeding, you need to answer three questions:
+
+- Which operating system to use for building cloud resources: macOS, Windows, or Linux?
+- Which cloud provider are you using? This guide covers AWS, Azure, and GCP.
+- Which programming language will you use?
```
[the remark this answers is not in the document]
Comment (spara) [reply content/blog/configuring-your-dev-environment/index.md:33]: added

File: content/blog/configuring-your-dev-environment/index.md [hunk 4]
```diff
@@ -0,0 +1,681 @@
...
+- **Code editor:** This is personal preference but make sure that it can perform code completion, error checking, and use enums. These features can be the difference between hitting `Tab` and searching through online documentation for a function.
+- **Pulumi:** You can install Pulumi with a package manager and configure
+
+## Decisions
+
+Before proceeding, you need to answer three questions:
+
+- Which operating system to use for building cloud resources: macOS, Windows, or Linux?
+- Which cloud provider are you using? This guide covers AWS, Azure, and Google Cloud.
+- Which programming language will you use. Pulumi supports Node.js (JavaScript and Typescript), Python 3.6 or higher, Golang, and .NET (C#, F#, and VB)?
```
Comment (davidwrede) [inline content/blog/configuring-your-dev-environment/index.md:33]:
```suggestion
- Which programming language will you use? Pulumi supports Node.js (JavaScript and Typescript), Python 3.6 or higher, Golang, and .NET (C#, F#, and VB).
```

**Review by davidwrede — APPROVED** [review]

**Pull request merged**